pkg/k8shandler/visualization.go

```diff
@@ -769,7 +765,7 @@ func newKibanaPodSpec(cluster *logging.ClusterLogging, kibanaName string, elasti
 
 func createSharedConfig(namespace, kibanaAppURL, kibanaInfraURL string) *v1.ConfigMap {
 	return NewConfigMap(
-		"sharing-config",
+		loggingSharedConfigMapName,
```

Comment:
Can we update `kibanaAppURL` -> `kibanaPublicAppURL`?  Not sure about `kibanaInfraURL`.

Reply:
I think it should be `kibanaAppPublicURL`